### PR TITLE
Add a query layer pointer if ngeoQueryOptions.hoverCursor is true

### PR DESCRIPTION
**Pull request by fredj**
replace #2134 

todo: test with an authenticated layer

### CODE REVIEW

File: src/services/layerHelper.js [hunk 1]
```diff
@@ -81,7 +81,8 @@ ngeo.LayerHelper.prototype.createBasicWMSLayer = function(sourceURL,
   const source = new ol.source.ImageWMS({
     url: sourceURL,
     params,
-    serverType: olServerType
+    serverType: olServerType,
+    crossOrigin: 'anonymous'
```
Comment (sbrunner) [inline src/services/layerHelper.js]: I think that we should add an option for that, and in a second time id should be configurable by the OGC server :-)

**Comment by rbovard** [comment]
> todo: test with an authenticated layer

@fredj Did you managed to test it? Do you think it won't work as it?

**Comment by rbovard** [comment]
Thanks @fredj!

**Comment by rbovard** [comment]
@fredj It was merged in master but which version was it on march (`2.1` or `2.2`)?

**Pull request merged**